### PR TITLE
Update PianoRoll.cpp

### PR DESCRIPTION
**Pull request by musikBear**
I have added one method
void PianoRoll::strumNotesDn()
This method will strum a stack of notes with the size of user-chosen value of Q
I have added one button for this action next to the Quantize button.
This button has tooltip, but it does not have an icon!
I hope our artists will create one in the style of current default theme. (if my patch is accepted.
The forward prototype declaration is commited alone in its own PR

Request code-review.

### CODE REVIEW

File: src/gui/editors/PianoRoll.cpp [hunk 1]
```diff
@@ -4177,6 +4177,96 @@ int PianoRoll::quantization() const
 	return DefaultTicksPerBar / Quantizations[m_quantizeModel.value() - 1];
 }
 
+int returnStrum(int st)//input > read value from comboBox Q. Output size of strum
+{
+int strumSize = 0;
+
+switch (st)
```
Comment (allejok96) [inline src/gui/editors/PianoRoll.cpp:4184]: This function can be replaced by `PianoRoll::quantization()`

File: src/gui/editors/PianoRoll.cpp [hunk 2]
```diff
@@ -4177,6 +4177,96 @@ int PianoRoll::quantization() const
...
+  m_pattern->addJournalCheckPoint();
+  NoteVector notes = getSelectedNotes();//NoteVector of class QVector
+  notesInChord = notes.count();//how many notes are in this chord
+  index = m_quantizeModel.value(); //uses UserSet Q-Value in combobox
+                                   //This value is position, so it need to be qualified
+                                   //to real piano/roll values
+  int strumSz = returnStrum(index); //size of note movement 
+  int strum = strumSz;//keep orr. value of selected strumming
+  bool firstDone = false;//first note should not be moved
+  for( Note* n : notes ) //for each selected
```
Comment (allejok96) [inline src/gui/editors/PianoRoll.cpp:4250]: strumSz, firstDone and notesInChord could be omitted for something like
```cpp
int count = notes.count()
for (int i=0; i<count; i++)
{
    n = notes.at(count - 1 - i)
    n->setPos(n->pos() + strum * i)
}
```

**Comment by allejok96** [comment]
Bear, the way you are using git have created trouble for yourself. Let me try to explain:

Imagine a git branch is like a copy of a paper. Your friend has written a story, and you want to change it. This is how it would look like:

1. Your friend sends you a `master` paper.
2. You make a *copy* of and call it `draft`, so you can always keep the `master` in your desk.
3. You make changes to the `draft`.
4. You send the `draft` to your friend so he can look at it.
5. If it looks good your friend will apply the changes from your `draft` to his `master` paper.

This is the problem you have created:

1. You got the `master` from LMMS.
2. You made a copy and called it `June2020mB`
3. You send `June2020mB` as a PR so the LMMS devs can look at it, but it doesn't get approved.
4. **Then at some point, you have applied (merged) your own `June2020mB` to your `master`**
5. Now your `master` is not the same as LMMS `master`.
5. When you make a new copy and call it `Envelope-for-Instruments`, it will be a dirty copy. It's like you are using the same paper to write on, instead of taking a new clean paper every time.
6. If you try to send it to LMMS it will be rejected because it includes both the changes for `Envelope-for-Instruments` and for `June2020mB`.

This situation can be fixed, but it is not possible to do with github alone.

File: src/gui/editors/PianoRoll.cpp (hunk 1 above)
Comment (musikBear) [reply src/gui/editors/PianoRoll.cpp:4184]: How? I use the manually set value by the user, to let user choose between all the Q-values that is in the Q-dropdown. 
How should `PianoRoll::quantization()` be able to give that functionality?

File: src/gui/editors/PianoRoll.cpp (hunk 2 above)
Comment (musikBear) [reply src/gui/editors/PianoRoll.cpp:4250]: Is that more readable?
Is there something wrong with the way i implemented it?

**Comment by musikBear** [comment]
> Bear, the way you are using git have created trouble for yourself. Let me try to explain:

It is more complicated. In my own local lmms in a virtual-box with Mint, i have made much more code.
Git will change my local lmms-code to Master's-code, if i use Git cmd to update my gitHub-repo.
**That** alone is the reason for my attempts to avoid updating through gitHub. 
I would *loose*all the other code i have locally!
And now i dare say i have succeeded.
My own local repo is untouched and i have made a PR against Master, with only the code i have made.

File: src/gui/editors/PianoRoll.cpp (hunk 1 above)
Comment (allejok96) [reply src/gui/editors/PianoRoll.cpp:4184]: The function you have created called `returnStrum()` returns the same value as the already present `quantization()`. They do the same thing.

**Pull request merged**